homeassistant/components/airnow/config_flow.py

```diff
@@ -98,6 +98,41 @@ async def async_step_user(self, user_input=None):
             errors=errors,
         )
 
+    @staticmethod
+    @core.callback
+    def async_get_options_flow(
+        config_entry: config_entries.ConfigEntry,
+    ) -> config_entries.OptionsFlow:
+        """Create the options flow."""
+        return OptionsFlowHandler(config_entry)
+
+
+class OptionsFlowHandler(config_entries.OptionsFlow):
```

Comment:
```suggestion
class OptionsFlowHandler(config_entries.OptionsFlowWithConfigEntry):
```

Reply:
This is per the documentation at https://developers.home-assistant.io/docs/config_entries_options_flow_handler... is there a different source of documentation?

Reply:
No, sometimes the doc's doesn't keep up.
Using `OptionsFlowWithConfigEntry` just provides you with the config entry so you can remove `__init__` as I have suggested below.